src/components/dashboard/main-content/course-enrollments/data/hooks.js

```diff
@@ -147,15 +177,13 @@ export const useCourseUpgradeData = ({
     };
 
     fetchData();
-  }, [
-    courseRunKey,
-    enterpriseId,
-    subscriptionLicense,
-    location,
-  ]);
+  }, [courseRunKey, enterpriseId, subscriptionLicense, location, couponCodes, courseRunPrice]);
```

Comment:
[confirmation] The addition of `couponCodes` and `courseRunPrice` don't make this `useEffect` run constantly, right? If `courseRunPrice` is included is set inside the `useEffect` function, would that make it trigger again?

Reply:
courseRunPrice shouldn't have been in here 🤔.